### ISSUE TITLE
Enable firebase Apple integration

### ISSUE THREAD

**Issue by ogallagher** [hoanghm/Proact#36]
Register an Apple apps (iOS, macOS) in the firebase project.

# Branches
- feature/firebase-apps

**Comment by ogallagher** [comment]
I made some progress as shown here https://github.com/hoanghm/Proact/issues/35#issuecomment-2184123535, but I've so far been unable to build and deploy an ios app in my local environment.

**Comment by hoanghm** [comment]
Do we need this since we're just creating an android app? I think this can be optional.

**Comment by ogallagher** [comment]
Yes, @hoanghm the github project priority is currently set to **optional**.
